app/views/tasks/show.html.slim

```diff
@@ -160,9 +160,26 @@
             = link_to @task.license.link, @task.license.link
           - else
             = t('.no_license')
-
+    - if @task.contribution?
+      legend.toggle-next.closed
+        span.number = '3'
+        => t('tasks.shared.contribution_info')
+        span.fa-solid.fa-caret-down
+      .show-table.exercise-show style="display:none;"
```

Comment:
I found another inline-style that sneaked into the code 🤓

Reply:
It is the same as found with the additional information.

The additional information will be fixed by #1527. I would wait until these changes are merged.

Reply:
Alright, fine for me. Before merging this PR, we should probably double-check that no further inline-style is left.

Reply:
With my latest push, I've fixed this occurrence and also validated that no other inline styles are currently used.